apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+
+## NFT
+
+### `token_activities_v2`
+
+_Has an aggregate view for summary data called `token_activities_v2_aggregate`_
+
+This table tracks token activities and is especially useful for tracking NFT activity. This includes both v1 and v2 data.
+
+| Field                      | Type    | Description                                                                                                                              |
+| -------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| after_value                | String  | The value of a token property after the transaction. Ex. "100"                                                                           |
+| aptos_names_from           | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| aptos_names_from_aggregate | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| aptos_names_to             | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| aptos_names_to_aggregate   | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| before_value               | String  | The value of a token property before the transaction. Ex. "50"                                                                           |
+| current_token_data         | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| entry_function_id_str      | String  | The identifier of the function called in this transaction. Ex. "transfer"                                                                |
+| event_account_address      | String  | This is an Aptos account address related to the event. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a"          |
+| event_index                | bigint  | Index of the event within the transaction. Ex. 1                                                                                         |
+| from_address               | String  | This is an Aptos account address from which the token was sent. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a" |
+| is_fungible_v2             | Boolean | Indicates whether the token is fungible. Ex. False for NFTs.                                                                             |
+| property_version_v1        | bigint  | The version of the token's properties under schema version 1. Ex. 1                                                                      |
```

Comment:
This field is only for token standard v1. It's always 0 for v2. It's more or less safe to ignore moving forward.